statcord/client.py

```diff
@@ -85,7 +85,10 @@ def servers(self):
 
     @property
     def users(self):
-        return str(sum(g.member_count for g in self.bot.guilds))
+        try:
+            return str(sum(g.member_count for g in self.bot.guilds))
+        except:
+            return len(self.bot.user)
```

Comment:
ClientUser has no __len__() maybe just return 0 ?

Reply:
Supposed to be self.bot.users I'm dumb.